src/ResourceManager/PolicyInsights/Commands.PolicyInsights/Commands.PolicyInsights.csproj

```diff
@@ -91,17 +91,16 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Management.PolicyInsights, Version=0.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.PolicyInsights.0.9.0-preview\lib\net452\Microsoft.Azure.Management.PolicyInsights.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.PolicyInsights, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.PolicyInsights.1.0.0\lib\net452\Microsoft.Azure.Management.PolicyInsights.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.2.3.10\lib\net452\Microsoft.Rest.ClientRuntime.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.2.3.11\lib\net452\Microsoft.Rest.ClientRuntime.dll</HintPath>
```

Comment:
Microsoft.Rest.ClientRuntime, Microsoft.Rest.ClientRuntime.Azure and Newtonsoft.Json shouldn't be in the .csproj or packages.config files since they are referenced elsewhere.

For example, every .csproj file should have a reference to the following file at the bottom of [their file](https://github.com/Azure/azure-powershell/blob/preview/tools/Common.Dependencies.targets)

Here's an example of a .csproj file referencing [this file]( https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Cdn/Commands.Cdn/Commands.Cdn.csproj#L165)